src/main.ts

```diff
@@ -16,48 +21,101 @@ async function commandExists(cmd: string): Promise<boolean> {
   return code === 0
 }
 
+async function runSbt(baseDir: string, pluginVersionInput: string): Promise<void> {
+  const projectDir = path.join(baseDir, 'project')
+
+  if (!fs.existsSync(projectDir)) {
+    core.setFailed(`${baseDir} is not a valid sbt project: missing folder '${projectDir}'.`)
+    return
+  }
+
+  const uuid = crypto.randomUUID()
+  const pluginFile = path.join(projectDir, `github-dependency-graph-${uuid}.sbt`)
+
+  const pluginVersion =
```

Comment:
There is some duplication in here, for example we do this here and also in the Mill section, but I honestly don't care that much since it's so small. Maybe if we ever add more we can clean this up more, but for now I don't think it's a huge deal if your'e alright with it.

Reply:
That seems fine